Build libfilezilla statically, notwithstanding its protests.

--- a/configure.ac
+++ b/configure.ac
@@ -43,10 +43,6 @@ if test "$locales" != "yes" -a "$locales
   AC_MSG_ERROR([invalid flags: --disable-locales cannot be used together with --enable-localesonly])
 fi
 
-if test "$enable_static" != "no"; then
-  AC_MSG_ERROR([libfilezilla does not support --enable-static])
-fi
-
 if ! test "$localesonly" = "yes"; then
 
 # Check if we're on Windows
@@ -145,6 +141,9 @@ if test "$windows" = "0" && test "$mac"
   CHECK_TCP_INFO
 fi
 
+# XtraDeb: Needed for linking a static libfilezilla
+libdeps="-lgnutls -lcrypt $libdeps"
+
 
 AC_SUBST([libdeps])
 
